example-config.yaml

```diff
@@ -40,6 +40,8 @@ slack:
     # Mapping between Project owners login and Slack users.
     akieras: kieras
     eduardo: dudu
+  # Cost minimum to notify
+  cost_min_to_notify: 0.0
```

Comment:
Move this to line 30.
Change comment to "Cost minimum to actually send a message (notify) to the owner."

Reply:
these changes are made in commit 19ae898